src/components/Game/Result/Result.js

```diff
@@ -1,36 +1,118 @@
-import React from 'react';
+import React, { Component } from 'react';
 import PropTypes from 'prop-types';
+import { Redirect } from 'react-router-dom';
 import { connect } from 'react-redux';
+import classNames from 'classnames';
 
-const ResultView = ({ words, answer }) => {
-    const isAnswerCorrect = () => words.includes(answer);
-
-    return (
-        <div>
-            {
-                (isAnswerCorrect())
-                    ? 'gratulacje, wygrałeś'
-                    : (
-                        `nie udało się, może następnym razem.
-                        poprawne słowa to:
-                        ${words
-                            .map((word) => word.toUpperCase())
-                            .join(', ')}`
-                    )
-            }
-        </div>
-    );
-};
+import classes from './Result.scss';
+import { submitAnswer } from '../../../store/actions/answer';
+
+class ResultView extends Component {
+    componentWillUnmount() {
+        const { dispatch } = this.props;
+        dispatch(submitAnswer(''));
+    }
+
+    isAnswerCorrect = () => {
+        const { words, answer } = this.props;
+        return words.includes(answer);
+    };
+
+    handlePlayAgainClick = () => {
+        const { history } = this.props;
+        history.replace('/game');
+    };
+
+    render() {
+        const { words, answer } = this.props;
+
+        let redirect = null;
+        if (!answer) {
+            redirect = <Redirect to="/game" />;
+        }
+
+        let other = (
+            <h6 className={classes.SectionHeader}>
+                Słowo nie ma anagramów
+            </h6>
+        );
+        if (words.length > 1 || (words.length === 1 && !this.isAnswerCorrect())) {
+            const otherItems = words
+                .filter((word) => word !== answer)
+                .map((word) => (
+                    <li className={classes.WordListItem} key={word}>
+                        {word.toUpperCase()}
+                    </li>
+                ));
+            other = (
+                <>
+                    <h6 className={classes.SectionHeader}>
+                        {this.isAnswerCorrect() ? 'Inne możliwe słowa:' : 'Poprawne słowa to:'}
+                    </h6>
+                    <ul className={classes.WordList}>{otherItems}</ul>
+                </>
+            );
+        }
+
+        return (
+            <main className={classes.Result}>
+                {redirect}
+
+                <div className={classes.AnswerSection}>
+                    <h6 className={classes.SectionHeader}>Twoje słowo:</h6>
```

Comment:
the website/document has a structure
the title goes in `h1` element
then sections can have `h2`s
then subsections may use `h3`
it all gives some sense of hierarchy of content
here usage of `h6` suggests you have used `h1`-`h5` headings before, somewhere higher in the structure
I'd recommend using semantically correct heading (perhaps `h2` in this case) and styling it according to your wishes

Reply:
👍